docs/reference/data-streams/lifecycle/apis/delete-lifecycle.asciidoc

```diff
@@ -4,7 +4,7 @@
 <titleabbrev>Delete Data Stream Lifecycle</titleabbrev>
 ++++
 
-Deletes the lifecycle from a set of data streams.
+Deletes the <<index-lifecycle-management,lifecycle>> from a set of data streams.
```

Comment:
is this the wrong link? AFAIK data streams have their own lifecycle management, so this would be referring to https://www.elastic.co/guide/en/elasticsearch/reference/current/data-stream-lifecycle.html

Reply:
```suggestion
Deletes the <<data-stream-lifecycle.html,lifecycle>> from a set of data streams.
```

Ah, that does appear to be true. Good catch. So this?

Reply:
no `html`, but yes 👍